### DISCUSSION TITLE
No output from RUN commands

### DISCUSSION THREAD

**Discussion by johnrhunt** [sherifabdlnaby/phpdocker#15] [Q&A] [answered]
I'm trying to debug changes I've made in the Dockerfile but can't get any output when I run `make build`

eg:
RUN echo "Hello world"

has no output (although the build process does output the literal line from the Dockerfile itself).

How can I enable output of commands when building?

**Comment by sherifabdlnaby** [comment, marked as answer]
@johnrhunt This is the Docker BUILDKIT default configuration, you need to pass an environment variable that makes buildkit print out all the logs.

It's `BUILDKIT_PROGRESS=plain`
so just run
```
export BUILDKIT_PROGRESS=plain
```
before building and you shall see all the logs from builds steps.
